App.js

```diff
@@ -35,6 +35,7 @@ import { NavigationActions } from 'react-navigation';
 import remoteConfig from '@react-native-firebase/remote-config';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import { QueryClient, QueryClientProvider } from 'react-query';
+import Instabug, { NetworkLogger } from 'instabug-reactnative';
```

Comment:
Consider hidding Instabug dependency behind a thin wrapper module, which would have methods for:
1) Initialization
2) MakePrivateView
3) Callbacks/hooks (setColorTheme, onNavigationStateChange)